### PR TITLE
Docker name

### PR DESCRIPTION
**Pull request by XuguangSong98**

### CODE REVIEW

**Comment by EngineersBox** [comment]
This isn't a branch off main. You need to create a branch off main and then add the changes to that. This branch contains changes from the other PR #12 . Can you create a branch from main and make the changes with that please?

**Review by EngineersBox — CHANGES_REQUESTED** [review]
This branch contains changes from the other PR #12 . You'll need to remove them, there should only be one file with changes.

**Comment by XuguangSong98** [comment]
> This branch contains changes from the other PR #12 . You'll need to remove them, there should only be one file with changes.

removed updates

File: minitask/new_import_data.py [hunk 1]
```diff
@@ -1,13 +1,12 @@
-import wikipedia, gzip, requests
-
+import wikipedia, gzip, threading
```
Comment (EngineersBox) [inline minitask/new_import_data.py]: **[Required]** Still got some changes here that shouldn't be part of the PR.. might have missed these when you were reverting changes on this branch. Gonna want to revert these too.

**Review by EngineersBox — CHANGES_REQUESTED** [review]

File: minitask/new_import_data.py (hunk 1 above)
Comment (XuguangSong98) [reply minitask/new_import_data.py]: reverted in the latest commit

Comment (EngineersBox) [reply minitask/new_import_data.py]: There are still changes there though. There should not be any changes to the `new_import_data.py` file. The fact that it even appears on this PR means that there are changes. It should be exactly the same as the `main` branch.

File: compose/opensearch/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,3 @@
+FROM opensearchproject/opensearch:latest
```
Comment (EngineersBox) [inline compose/opensearch/Dockerfile:1]: **[Note]** Good catch re-adding this. Must've been deleted in a previous PR by accident 👍🏼

**Comment by EngineersBox** [comment]
Bumping this PR @alooba1

**Review by EngineersBox — APPROVED** [review]
LGTM
